components/compass/button/button.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+    leftIconGlyph?: string;
+    rightIconGlyph?: string;
+    style: string;
+    size: string;
+    label: string;
+    link?: string;
+    fullWidth?: boolean;
+    disabled?: boolean;
+}
+
+const ButtonCommon: React.FC<Props> = ({leftIconGlyph, rightIconGlyph, style, size, label, link, fullWidth, disabled}: Props) => {
+    const anchorTag = (
+        <a
+            href={link}
+            className={classNames([`Button Button___${style} Button___${size}`, {fullWidth}])}
```

Comment:
We'll need default values for `style` and `size` in case one isn't provided via props.